### PR TITLE
Add --input_variant parameter for handling single variant

### PR DESCRIPTION
**Pull request by jylee-bcm**

### CODE REVIEW

**Comment by hyunhwan-jeong** [comment]
Can you find why the smoke test failed. @jylee-bcm

File: bin/generate_input_vcf.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+vcf_template = string.Template("""
+##fileformat=VCFv4.2
+##FORMAT=<ID=AD,Number=.,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth (reads with MQ=255 or with bad mates are filtered)">
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification">
+##FORMAT=<ID=SB,Number=4,Type=Integer,Description="Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias.">
+##FILTER=<ID=PASS,Description="All filters passed">
+##reference=file:///staging/human/reference/b37/b37.fa.default/reference.bin
```
Comment (hyunhwan-jeong) [inline bin/generate_input_vcf.py]: does it only handle hg19? not hg38?

File: main.nf [hunk 2]
```diff
@@ -27,53 +27,67 @@ def showVersion() {
...
+        println("To see usage and available parameters run `nextflow run main.nf --help`")
+        exit 1
     }
 }
 
 showUsage()
 showVersion()
 validateInputParams()
 
+process GENERATE_INPUT_VCF {
```
Comment (hyunhwan-jeong) [inline main.nf:150]: no need to process for both hg19/hg38?

File: nextflow.config [hunk 3]
```diff
@@ -1,4 +1,7 @@
 params {
+    input_vcf = null
+    input_variant = null
+
     run_id = 1
     ref_ver = "hg19"
     ref_dir = " "
```
Comment (hyunhwan-jeong) [inline nextflow.config]: better to do `null` instead of `" "`

**Review by hyunhwan-jeong — CHANGES_REQUESTED** [review]

File: bin/generate_input_vcf.py (hunk 1 above)
Comment (jylee-bcm) [reply bin/generate_input_vcf.py]: it just works regardless of it.
It even works without the line.
would you prefer to remove it?

File: main.nf (hunk 2 above)
Comment (jylee-bcm) [reply main.nf:150]: it just works regardless of that.

File: nextflow.config (hunk 3 above)
Comment (jylee-bcm) [reply nextflow.config]: I will apply that

**Comment by jylee-bcm** [comment]
> Can you find why the smoke test failed. @jylee-bcm

we can't see the error message but i am pretty sure that's related with the docker version problem.

**Review by hyunhwan-jeong — APPROVED** [review]

**Review by hyunhwan-bcm — APPROVED** [review]
LGTM

**Pull request merged**